setup.py

```diff
@@ -904,6 +894,7 @@ def filter_files(path_obj, all_files, allowed_files, disallowed_files):
             commands = {
                 "clang-format": ["--dry-run", "--Werror", "-i"] + c_files,
                 "black": ["--check", "--diff"] + python_directories,
+                "codespell": [],  # Codespell configuration is in setup.cfg 
                 # Test directory has too much pylint warning for now
                 "pylint": ["src_py"],
             }
```

Comment:
`codespell` should be added in `commands` in the else clause below, too, I think.